src/webui/webapplication.cpp

```diff
@@ -212,12 +208,12 @@ void WebApplication::translateDocument(QString &data)
             const QString sourceText = regexMatch.captured(1);
             const QString context = regexMatch.captured(3);
 
-            QString translation = sourceText;
-            if (isTranslationNeeded) {
-                const QString loadedText = m_translator.translate(context.toUtf8().constData(), sourceText.toUtf8().constData(), nullptr, 1);
-                if (!loadedText.isEmpty())
-                    translation = loadedText;
-            }
+            const QString loadedText = m_translator.isEmpty()
```

Comment:
It's not the same as I meant. See:
```c++
bool QTranslator::isEmpty() const
{
    Q_D(const QTranslator);
    return !d->messageArray && !d->offsetArray && !d->contextArray
            && d->subTranslators.isEmpty();
}
```
Wouldn't it be better to keep last translation loading state instead (just one boolean variable)?

Reply:
Added `bool m_translationFileLoaded` to track it.